lib/rollbar/notifier.rb

```diff
@@ -675,34 +676,41 @@ def ssl_verify_mode
       end
     end
 
-    def write_item(item)
-      if configuration.use_async
-        @file_semaphore.synchronize do
-          do_write_item(item)
-        end
-      else
-        do_write_item(item)
-      end
-    end
-
     def do_write_item(item)
```

Comment:
Is there a reason other than style for the method rename `write_item` -> `do_write_item`?

Reply:
I misread that. I see it now.